.rubocop.yml

```diff
@@ -28,6 +28,9 @@ Style/FormatStringToken:
   Exclude:
     - spec/**/*
 
+Style/HashEachMethods:
+  Enabled: false
```

Comment:
This setting looks unnecessary.

Reply:
I had a build failing that prompted me to add it here. Can you tell me more about why this looks unnecessary?

Reply:
By default, it is better to use the setting in config/default.yml.

Reply:
I think I'm getting confused around what `pending` will do. It appears that breaks builds also. Are you suggesting to leave it as `pending` and then remove it from here?

Reply:
Yeah, config/default.yml leaves the specification of `Enabled: pending`. OTOH, I expect this setting to be removed from .rubocop.yml.

> It appears that breaks builds

Can you show me what build error is occurring?